src/app/views/assignments/popups/questions-list/questions-list.component.ts

```diff
@@ -100,7 +104,16 @@ export class QuestionslistComponent implements OnInit, AfterViewInit {
     });
   }
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    const testSearchObj: SearchQuestionPaperVM = new SearchQuestionPaperVM ("1",1000);
+
+    this.testConfigService.getAllQuestionPaper(testSearchObj).subscribe(resp => {
+        resp.tests.forEach(element => {
```

Comment:
This information of test name should be in the default question search api . We do not need to make another call for getting test name .

there is a separate attribute `usedInTestDetails` for the same in the question object.